lib/routes/users.js

```diff
@@ -156,7 +156,9 @@ router.get('/:id/recs/genres/:pageNumber', async (request, response) => {
       userId,
       pageNumber
     );
-    response.send({ recs, maxPageReached });
+    if (recs) {
+      response.send({ recs, maxPageReached });
+    } else response.status(400).send('No shows in watchlist');
```

Comment:
What happens when no recs are found?
I would expect an empty array.
An empty array is truthy, thus, you have to check the length.

`400` means `bad request`. But maybe the request is correct. 
If no results are found, `404` is the correct response.

Reply:
when no recs are found i am returning an empty object (without recs and maxPageReached). since then recs won't exist/be null this is empty. because recs are paginated so it returns an array and the maxPagedReached. I changed the response status